### PR TITLE
docs(badge): add badge to migration guide, and blog post

### PR DESCRIPTION
**Pull request by braddialpad**
Adding the recent badge breaking changes to the migration guide and wrote a blog post on some of our new updates.

### CODE REVIEW

File: docs/about/whats_new/posts/2022-12-9.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+- [Badge](https://dialpad.design/components/badge.html)
+- [Badge Vue Component](https://vue.dialpad.design/?path=/story/components-badge--default)
+
+## Badge
+
+The recent updates to badge cause breaking changes in Dialtone v7.9.0 and Dialtone Vue v2.43.0 and Dialtone Vue 3 v3.29.0
+
+The migration has been performed for all existing `DtBadge` components and `d-badge` classes in Dialpad and Dialpad Meetings. However if you have an existing feature branch or are in a project outside of Dialpad / Dialpad Meetings that involves badge, the changes will not have been made there so you'll have to make the updates yourself. If you need to do this, please follow the badge section of the [migration guide](https://github.com/dialpad/dialtone/blob/staging/migration_guide/MIGRATION_GUIDE.md#badge).
+
+### Label
```
Comment (francisrupert) [inline docs/about/whats_new/posts/2022-12-9.md]: Might be good to have the raw variants listed here too. 

image

I can see how someone might be confused by the examples.

File: migration_guide/MIGRATION_GUIDE.md [hunk 2]
```diff
@@ -531,7 +531,48 @@ Remove any usage of `d-svg` and `d-svg--*`.
...
+
+#### Badge
+
+##### DtBadge Vue Component
+
+The "color" prop has been removed from the DtBadge component and replaced with "type".
+
+##### Styling
+
+All `d-badge` color classes are no longer valid and will have to be replaced with a "type" class. There is no direct mapping for this so you'll have to decided which works best on a case by case basis. Speak with your designer if you are unsure.
```
Comment (francisrupert) [inline migration_guide/MIGRATION_GUIDE.md]: **Suggested update:**

> The previous `d-badge` color classes are no longer valid and will have to be replaced with a "type" class. There is no direct mapping guide, as the previous use of badge colors may not imply which updated type to use. Use your judgment on a case-by-case basis and validate with your Product Designer.

**Review by francisrupert — COMMENTED** [review]
Nice. Just a few reactions.

File: docs/about/whats_new/posts/2022-12-9.md (hunk 1 above)
Comment (braddialpad) [reply docs/about/whats_new/posts/2022-12-9.md]: Yeah good idea, added.

File: migration_guide/MIGRATION_GUIDE.md (hunk 2 above)
Comment (braddialpad) [reply migration_guide/MIGRATION_GUIDE.md]: thanks! updated.

**Pull request merged**